src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -171,14 +171,6 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
             }
         });
 
-        add(new GeneralAction(Actions.MERGE_DOI, Localization.lang("Get BibTeX data from DOI")) {
```

Comment:
I so prefer it to stay here (as well). This has been one of my most used features recently.